src/shared/components/formik-fields/EditableLabelField.tsx

```diff
@@ -53,7 +53,7 @@ const EditableLabelField: React.FC<EditableLabelFieldProps> = ({
         </Flex>
       ) : (
         <div className="hacDev-editable-label-field__label">
-          {value} {!label && editIcon}
+          {value} {editIcon}
```

Comment:
Target port doesn't seem to be showing edit icon anymore:
(`editIcon` is also used in line 36)
![0](https://user-images.githubusercontent.com/20013884/156376541-edcd4bbe-990a-4f63-8ce5-0e2409570c19.png)

Reply:
fixed